app/src/main/java/org/kiwix/kiwixmobile/nav/destination/reader/KiwixReaderFragment.kt

```diff
@@ -67,6 +67,7 @@ import java.io.File
 private const val HIDE_TAB_SWITCHER_DELAY: Long = 300
 
 class KiwixReaderFragment : CoreReaderFragment() {
+  private var isFullScreen: Boolean = false
```

Comment:
maybe `isFullscreenVideo`, so as not to cause confusion with the separate fullscreen feature